src/Permissions/Backend/PermissionManager.vala

```diff
@@ -36,95 +36,59 @@ public class Permissions.Backend.PermissionManager {
-        //      "video-display"
-        //  ));
-        //  insert ("sockets=wayland", new PermissionDescription (
-        //      _("Access Wayland windowing system"),
-        //      _("App is able to show graphical windows"),
-        //      "video-display"
-        //  ));
-        insert ("sockets=pulseaudio", new PermissionDescription (
-            _("Sounds"),
-            _("App may be able to play sounds"),
-            "preferences-desktop-sound"
-        ));
-        insert ("sockets=system-bus", new PermissionDescription (
-            _("D-Bus system bus"),
-            _("App may be able to talk to, control and read information from system components via D-Bus"),
-            "internet-chat"
-        ));
-        insert ("sockets=session-bus", new PermissionDescription (
-            _("D-Bus session bus"),
-            _("App may be able to talk to, control and read information from other apps via D-Bus"),
-            "internet-chat"
-        ));
-        insert ("sockets=ssh-auth", new PermissionDescription (
-            _("Secure Shell agent"),
-            _("App may be able to access other devices on the network via SSH"),
-            "utilities-terminal"
-        ));
-        insert ("sockets=cups", new PermissionDescription (
-            _("Print"),
-            _("App may be able to access printers"),
-            "printer-printing"
+
+        insert ("filesystems=home", new PermissionDescription (
+            _("Home Folder"),
+            _("Access your entire home folder"),
+            "user-home"
         ));
-        insert ("devices=dri", new PermissionDescription (
-            _("GPU acceleration"),
-            _("App may be able to accelerate graphical output"),
-            "applications-graphics"
+
+        insert ("filesystems=host", new PermissionDescription (
+            _("System Folders"),
+            _("Access system folders, not including the operating system or system internals"),
+            "drive-harddisk"
         ));
+
         insert ("devices=all", new PermissionDescription (
             _("Devices"),
-            _("App may be able to access devices like webcams"),
+            _("Access devices like webcams and microphones"),
             "accessories-camera"
         ));
-        insert ("filesystems=host", new PermissionDescription (
-            _("System directories"),
-            _("App may be able to access system directories"),
-            "drive-harddisk"
+
+        insert ("sockets=pulseaudio", new PermissionDescription (
+            _("Sounds"),
+            _("Play sounds"),
+            "preferences-desktop-sound"
         ));
-        insert ("filesystems=home", new PermissionDescription (
-            _("Home directory"),
-            _("App may be able to access your home directory"),
-            "user-home"
+
+        insert ("shared=network", new PermissionDescription (
+            _("Network"),
+            _("Access the Internet and local networks"),
```

Comment:
This might also include networking hardware information, i.e. the device's MAC address which could be used for tracking/fingerprinting.